geometry_msgs/Wrench[] wrench_list
string[] name_col1
string[] name_col2
